Gems/MiniAudio/Code/Source/Clients/MiniAudioListenerComponentConfig.h

```diff
@@ -27,5 +27,14 @@ namespace MiniAudio
         AZ::EntityId m_followEntity;
 
         AZ::u32 m_listenerIndex = 0;
+
+        //! Inner cone angle
+        float m_innerAngleInRadians = 3.f/5.f * AZ::Constants::TwoPi;
+        float m_innerAngleInDegrees = m_innerAngleInRadians * 360.f/AZ::Constants::TwoPi;
+        //! Outer cone angle
+        float m_outerAngleInRadians = 1.5f * AZ::Constants::Pi;
+        float m_outerAngleInDegrees = m_outerAngleInRadians * 360.f/AZ::Constants::TwoPi;
+        //! Volume outside of outer cone
+        float m_outerVolume = 50.0f;
```

Comment:
These initial values are duplicated in MiniAudioPlaybackComponentConfig.h , consider consolidating them in a shared header if you're going to have to change them in multiple places.

Reply:
The default values I've assigned in MiniAudioListenerComponentConfig.h and MiniAudioPlaybackComponentConfig.h are actually different. For the playback component I have both inner and outer angles set to 2 * pi, for the listener I have the inner set to 3/5 * 2 * pi and the outer set to 1.5 * pi. In the case of `m_outerVolume`, I have it set to 0% for the playback component and 50% for the listener.